src/mlpack/methods/time_series_forecast/exponential_smoothing/ses.hpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file methods/time_series/exponential_smoothing/ses.hpp
+ * @author Aditi Pandey
+ *
+ * Single exponential smoothing.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_EXPONENTIAL_SMOOTHING_SES_HPP
+#define MLPACK_METHODS_EXPONENTIAL_SMOOTHING_SES_HPP
+
+#include <mlpack/prereqs.hpp> 
+#include <armadillo>
```

Comment:
```suggestion

```
This is not required. You only need `<mlpack/prereqs.hpp>` here.

Reply:
Okay 👍🏼, I'll remove that